code/modules/admin/verbs/honksquad.dm

```diff
@@ -133,7 +133,7 @@ GLOBAL_VAR_INIT(sent_honksquad, 0)
 
 	var/obj/item/card/id/W = new(src)
 	W.name = "[real_name]'s ID Card"
-	W.icon_state = "centcom_old"
```

Comment:
This was removed because its only used in one spot, and it doesnt make sense. Clowns should have clown IDs, not centcomm ones.